docs/topics/modules/reviewing_your_staged_application.adoc

```diff
@@ -0,0 +1,26 @@
+[id="reviewing_your_staged_application"]
+= Reviewing your staged application
+
+When you create a new codebase, the selected pipeline builds your application. It pushes version 1.0.1 of your new application into the _Stage_ environment, and then awaits your approval to deploy into the _Run_ environment.
```

Comment:
I'm not sure I'm reading this correctly, but it sounds like the pipeline is somehow the subject of this part. Does "It" refer to the pipeline? I'm not sure the pipeline itself is doing anything, it's just a pipeline type that the application uses to deploy to stage or run. Can we reword this to reflect this?

Reply:
So far whatever I have read and understood from my discussions with Devs seems to suggest that the Pipeline triggers/pushes these actions.
Some examples: 
Openshift docs: https://docs.openshift.com/container-platform/3.9/dev_guide/dev_tutorials/openshift_pipeline.html#overview
"This example demonstrates how to create an OpenShift Pipeline that will build, deploy, and verify a Node.js/MongoDB application using the nodejs-mongodb.json template."

Our UG docs (not authored by me): https://docs.openshift.io/user-guide.html#selecting_a_pipeline_type, Point #1 (On a totally tangential thought, wonder why this is a numbered list?): "It provides an end to end pipeline that moves your application through all the stages of application development, that is, build your source code, test your changes, rollout to the Stage environment, review, manually approve, and promote the changes to the Run environment."

I am not sure I understand your second last sentence, are you suggesting the application is the actor here?

But yet, here's an attempt at rewording it: 
When you create a new codebase, the selected pipeline pushes version 1.0.1 of your new application into the _Stage_ environment, and then awaits your approval to deploy into the _Run_ environment.

Please note, this was already existing in the docs>[Section 5.6](https://docs.openshift.io/getting-started-guide.html#working_with_pipelines) just above the adominition at the end of the module.

Reply:
I see. That sounds really odd to me in non technical terms to have a pipeline be the actor but if it's intentional and accurate, sounds like a non issue.

Reply:
I see. That sounds really odd to me in non technical terms to have a pipeline be the actor but if it's intentional and accurate, sounds like a non issue.